internal/pkg/instrumentation/bpf/database/sql/probe.go

```diff
@@ -27,6 +29,9 @@ const (
 
 	// IncludeDBStatementEnvVar is the environment variable to opt-in for sql query inclusion in the trace.
 	IncludeDBStatementEnvVar = "OTEL_GO_AUTO_INCLUDE_DB_STATEMENT"
+
+	// IncludeDBOperationEnvVar is the environment variable to opt-in for sql query operation in the trace.
+	IncludeDBOperationEnvVar = "OTEL_GO_AUTO_INCLUDE_DB_OPERATION"
```

Comment:
This will need to be added here: https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/main/docs/configuration.md#instrumentation-options

Reply:
updated